src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const lowerCaseCompare = query.toLowerCase();
+
+    return (
+      title.toLowerCase().includes(lowerCaseCompare)
+      || description.toLowerCase().includes(lowerCaseCompare)
```

Comment:
if you wrote the same code twice you need to create a function for this code and write it down before the component if it doesn't receive any state variables.